dandiapi/api/views/asset.py

```diff
@@ -124,6 +124,19 @@ def update(self, request, **kwargs):
         serializer = AssetDetailSerializer(instance=asset)
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+    # @permission_required_or_403('owner', (Dandiset, 'pk', 'version__dandiset__pk'))
+    def destroy(self, request, **kwargs):
```

Comment:
why `destroy` and not `delete` for consistency?

Reply:
`django-rest-framework` defines these ViewSet classes that correspond to REST "entities". The `list` method is mapped to `GET assets/`, `get` to `GET assets/{uuid}`, and `destroy` to `DELETE assets/{uuid}`.